Formula/pinocchio.rb

```diff
@@ -0,0 +1,42 @@
+class Pinocchio < Formula
+  desc "Efficient and fast C++ library implementing Rigid Body Dynamics algorithms"
+  homepage "https://stack-of-tasks.github.io/pinocchio"
+  url "https://github.com/stack-of-tasks/pinocchio/releases/download/v2.6.10/pinocchio-2.6.10.tar.gz"
+  sha256 "4ded587192561d434d4158fdae56dd36b0c3a0db65ae4768df94428161bad522"
+  license "BSD-2-Clause"
+  head "https://github.com/stack-of-tasks/pinocchio.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "boost"
+  depends_on "eigen"
+  depends_on "eigenpy"
+  depends_on "hpp-fcl"
+  depends_on "python@3.10"
+  depends_on "urdfdom"
+
+  def install
+    if build.head?
+      system "git", "submodule", "update", "--init"
+      system "git", "pull", "--unshallow", "--tags"
+    end
+
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DPYTHON_EXECUTABLE=#{Formula["python@3.10"].opt_libexec/"bin/python"}",
+                    "-DBUILD_UNIT_TESTS=OFF",
+                    "-DBUILD_WITH_COLLISION_SUPPORT=ON",
+                    *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    python_exe = Formula["python@3.10"].opt_libexec/"bin/python"
+    system python_exe, "-c", <<~EOS
```

Comment:
Is it just a python library? Or is that the only thing we're testing?

Reply:
This is a C++ lib with Python bindings.
We are only testing the Python bindings, which allows to certify the whole library is well compiled and installed.